lib/jekyll/commands/new.rb

```diff
@@ -11,6 +11,7 @@ def init_with_program(prog)
 
             c.option "force", "--force", "Force creation even if PATH already exists"
             c.option "blank", "--blank", "Creates scaffolding but with empty files"
+            c.option "skip_bundle", "--skip_bundle", "Skip 'bundle install'"
```

Comment:
I would prefer `--skip-bundle` here. Consistency of hyphens seems more intuitive.